src/world/area_mgm/mgm_02/mgm_02_0_header.c

```diff
@@ -1,8 +1,14 @@
 #include "mgm_02.h"
 
 #if VERSION_PAL
-s32 N(get_tattle)(void);
-INCLUDE_ASM(void, "world/area_mgm/mgm_02/mgm_02_0_header", mgm_02_get_tattle);
+extern s32 mgm_02_pal_variable;
```

Comment:
This can probably be defined/used using `N(pal_variable)`, and the declaration can be moved to a header.